contrib/kube-prometheus/manifests/prometheus/prometheus-k8s-service-monitor-kubelet.yaml

```diff
@@ -10,6 +10,9 @@ spec:
   - port: http-metrics
     interval: 30s
     honorLabels: true
```

Comment:
This can be removed now as it only was needed for cAdvisor metrics.
Those being exposed on separate endpoints now is actually good from a monitoring perspective.